pkg/ha/testdata/source.yaml

```diff
@@ -0,0 +1,20 @@
+server-id: 101
```

Comment:
If this config is used only once, I prefer to generate it in the unit test.

Reply:
fixed in https://github.com/pingcap/dm/pull/1910/commits/9e886e3cea8ce09168ecc6adc1cdbeb21ff7f6d6